### ISSUE TITLE
name of song shows in place of artist

### ISSUE THREAD

**Issue by kapijawastaken** [z-huang/music#1397]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions/workflows/build.yml).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

1. Search for any song
2. This appears on some songs (like Murder in my Mind by KORDHELL)

### Expected behavior

The artist's name should be there, instead of the song's.

### Actual behavior

The song's name is there, instead of the artist's.

### Screenshots/Screen recordings

![Screenshot_20240816_132107_InnerTune](https://github.com/user-attachments/assets/63c26eb2-f486-4c27-922e-a6ff5a476ce0)

### Logs

i do not have access to adb right now since i dont have access to a computer right now.

### InnerTune version

0.5.7

### Android version

Android 13

**Comment by z-huang** [comment]
Does it still happen? Mine's working good.

**Comment by kapijawastaken** [comment]
yep![Screenshot_20240827_173724_InnerTune.png](https://github.com/user-attachments/assets/4dc0f13b-9d21-41f3-a289-787523c9b190)

**Comment by z-huang** [comment]
Are you using debug build? Have you tried refetching the song?

**Comment by kapijawastaken** [comment]
i was using 0.5.8 at the time of the last screenshot, now i upgraded it to 0.5.9, and i still had the issue until i refetched the song
